server/authentication/authInit.js

```diff
@@ -5,12 +5,13 @@ const logger = require('../../log');
 module.exports = function(signInService, userService, audit) {
 
     async function localInit(username, password, done) {
+        const capUsername = username.toUpperCase();
```

Comment:
confused as to why you are changing this function.  Shouldn't it just be deleted as all logins should go through auth?

Reply:
Yes everything that a user sees. It was just for instances when running locally with local signin form